superset-frontend/packages/superset-ui-core/package.json

```diff
@@ -55,7 +55,7 @@
     "math-expression-evaluator": "^1.3.8",
     "pretty-ms": "^7.0.0",
     "react-error-boundary": "^1.2.5",
-    "react-markdown": "^4.3.1",
+    "react-markdown": "^7.1.1",
```

Comment:
Looks like this one is bumped to too high of a version, causing some (if not all) of the CI failures.